src/KubernetesClient/KubernetesClient.csproj

```diff
@@ -7,7 +7,6 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="7.0.0" />
```

Comment:
see #1534

Reply:
The `UpDownCounter` introduced in 7.0, `net6.0` framework reference does not have this reference, so we have to reference the package at least for `net6.0`, maybe a condition could be added like `Condition="'$(TargetFramework)'='net6.0'"`, safe to remove for `net8.0`

Reply:
good pickup

Reply:
seemed fix not pushed